packages/tauri/build-tauri-conf.js

```diff
@@ -0,0 +1,40 @@
+const { resolve } = require("path");
```

Comment:
This is a new file that takes care of making sure `tauri` builds have the vendor `version`, `name`, and a few other things (`author`, for example). It updates the `tauri.conf.json` and the `Cargo.toml` files.